### PR TITLE
Signin with checkbox

### PR DESCRIPTION
**Pull request by P3nny**
Fixes: https://github.com/railslove/recover-backlog/issues/2

We're choosing an explicit variant of linking to the contract.

### CODE REVIEW

**Comment by P3nny** [comment]
Please delete the contract.pdf and move the link to the word "Nutzungsvertrag"

File: pages/business/setup/signup.tsx [hunk 1]
```diff
@@ -28,6 +30,10 @@ const LoginSchema = Yup.object().shape({
   confirmPassword: Yup.string()
     .required('Passwordwiederholung muss angegeben werden.')
     .oneOf([Yup.ref('password'), null], 'Passwörter stimmen nicht überein.'),
+  confirmContract: Yup.bool().oneOf(
+    [true],
+    'Sie müssen dem Vertrag zustimmen.'
+  ),
```
Comment (salimhb) [inline pages/business/setup/signup.tsx:36]: This is blocking users who are not part of `isCareEnv` or `isHealthEnv` from registering as they don't have the `Checkbox` to confirm.

**Review by salimhb — CHANGES_REQUESTED** [review]

File: pages/business/setup/signup.tsx [hunk 2]
```diff
@@ -88,6 +94,7 @@ const SetupSignupPage: React.FC<WithOwnerProps> = () => {
           email: '',
           password: '',
           confirmPassword: '',
+          confirmContract: false,
```
Comment (salimhb) [inline pages/business/setup/signup.tsx:97]:
```suggestion
          confirmContract: !isCareEnv && !isHealthEnv,
```
This should allow other users to register

**Review by salimhb — CHANGES_REQUESTED** [review]

**Review by salimhb — APPROVED** [review]

**Pull request merged**